### ISSUE TITLE
onSIPInfoReceived callback with null content

### ISSUE THREAD

**Issue by joaovirgilicw** [voximplant/flutter_voximplant#20]
I'm receiving null content on `onSIPInfoReceived` callback. `type` and `headers` are fine.

**Comment by VladimirBrejcha** [comment]
Hello @joaovirgilicw,

Please provide logs, it is desirable to run your application from XCode

**Comment by joaovirgilicw** [comment]
Hey @VladimirBrejcha, thanks for the response.

Here are the logs running from XCode

![Screen Shot 2021-03-24 at 15 39 33](https://user-images.githubusercontent.com/78375092/112366186-45a0c500-8cb7-11eb-9f6b-596521f1ad29.png)

First one is native and it's fine. The second one is on flutter and content is null.
My code is like this
```
call.onSIPInfoReceived = (call, type, content, headers) {
  print("[onSIPInfoReceived]: ($type, $content, $headers");
};
```

**Comment by VladimirBrejcha** [comment]
@joaovirgilicw Thanks for the contribution! Released in 2.6.1
